browser/src/control/jsdialog/Widget.PageSizeEntry.ts

```diff
@@ -0,0 +1,84 @@
+// @ts-strict-ignore
```

Comment:
please do not add this line to a new file, it was meant to ignore code formatting in the old ones